planner/core/physical_plans.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/pingcap/tidb/table"
 	"github.com/pingcap/tidb/types"
 	"github.com/pingcap/tidb/util/ranger"
+	"github.com/pingcap/tipb/go-tipb"
```

Comment:
order?

Reply:
It has been sorted, otherwise ci wouldn't pass :)